#!/usr/bin/env python
PACKAGE = "tick_calibration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_tick", double_t, 0, "An Integer parameter", 0.004, 0, 0.01)
gen.add("max_tick", double_t, 0, "A double parameter", 0.007, 0, 0.01)
gen.add("min_length", double_t, 0, "A double parameter", 1.0, 0, 10.0)
gen.add("min_ticks", int_t, 0, "A double parameter", 200, 0, 10000)
gen.add("buffer_size", int_t, 0, "A double parameter", 1000, 1, 10000)
gen.add("calibration_frequency", int_t, 0, "A double parameter", 1, 1, 100)

exit(gen.generate(PACKAGE, "tick_calibration", "TickCalibration"))
